README.md

```diff
@@ -328,6 +328,38 @@ to match test name.
 => (#:greenlight.test{:description ",,,", :title "test-2", :ns user, :line 5, :steps []})
 ```
 
+### Parallel Test Execution
+
+Tests can be executed in parallel by providing a `--multithread` option
```

Comment:
Naming suggestion: maybe call this `--parallel` instead of `--multithread`? It's a bit easier to remember and more closely captures the intent instead of being tied to the implementation.